content/source/docs/cloud/run/manage.html.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: "cloud"
+page_title: "Viewing and Managing Runs - Runs - Terraform Cloud"
+---
+
+# Viewing and Managing Runs
+
+Each workspace in Terraform Cloud includes a list of its current, pending, and historical runs. You can view information about these runs in the UI, and sometimes interact with them.
+
+Additionally, you can lock workspaces to temporarily prevent new runs.
+
+## Navigating Runs
+
+-> **API:** See [the Runs API](../api/run.html).
+
+Each workspace has two ways to view and navigate runs:
+
+- A "Runs" link, which goes to the full list of runs.
+- A "Current Run" link, which goes to the most recent active run. (This might not be the most recently initiated run, since runs in the "pending" state remain inactive until the current run is completed.)
+
+![runs list](./images/runs-list.png)
+
+From the list of runs, you can click to view or interact with an individual run.
+
+### The Run Page
+
+An individual run page shows the progress and outcomes of each stage of the run.
+
+![a run page](./images/runs-run-page.png)
+
+Most importantly, it shows:
+
+- The current status of the run.
+- The code commit associated with the run.
+- How the run was initiated, when, and which user initated it (if applicable).
+- A timeline of events related to the run.
+- The output from both the `terraform plan` and `terraform apply` commands, if applicable. You can hide or reveal these as needed; they default to visible if the command is currently running, and hidden if the command has finished.
+
+## Interacting with Runs
+
+-> **API:** See [the Runs API](../api/run.html).
+
+In workspaces where you have write permissions, run pages include controls for interacting with the run at the bottom of the page. Depending on the state of the run, the following buttons might be available:
+
+Button              | Available when:
+--------------------|----------------
+Add Comment         | Always.
+Confirm & Apply     | A plan needs confirmation.
+Override & Continue | A soft-mandatory policy failed (only available for owners team).
+Discard Run         | A plan needs confirmation or a soft-mandatory policy failed.
+Cancel Run          | A plan or apply is currently running.
+Force Cancel Run    | A plan or apply was canceled, but something went wrong and Terraform Cloud couldn't end the run gracefully (only available with workspace admin permissions).
+
+If a plan needs confirmation (with [manual apply](../workspaces/settings.html#auto-apply-and-manual-apply) enabled) or a soft-mandatory policy failed, the run will remain paused until a user with appropriate permissions uses these buttons to continue or discard the run. For more details, see [Run States and Stages][].
```

Comment:
The run states and stages link doesn't render right.

Reply:
Ack, I left the link reference in the page I moved the text from. Good catch!